src/components/makeResponsive.js

```diff
@@ -17,7 +17,7 @@ import {
     CLASS_IS_MEDIUM,
 } from '../constants';
 
-type Props = {
+type PropsShape = {
```

Comment:
Whats this?

Reply:
The class properties transform throws a fit unless the function is typed with `Props: { whatever }`. This was the simplest way to keep the function typed without throwing errors on compile. I can change the name if you have an alternative suggestion.